.github/workflows/Build.yaml

```diff
@@ -193,9 +192,45 @@ jobs:
           heap-size: 600M
           script: ./gradlew connectedDemoDebugAndroidTest --daemon
 
+      - name: Run local tests (including Roborazzi) for the combined coverage report (only API 26)
+        if: matrix.api-level == 26
+        run: ./gradlew testDemoDebugUnitTest -Proborazzi.test.verify=true # Add Prod if we ever add JVM tests for prod
```

Comment:
Is `-Proborazzi.test.verify=true` still needed with the property in `gradle.properties`?

Reply:
No, and I want to remove one of them. Thoughts? 

I decided to add it to `gradle.properties` because without them I noticed that the screenshot tests are still run on Robolectric, but no screenshots are taken or verified, so it's a waste of time. We might as well verify,

Reply:
I think `gradle.properties` is better, to always be verifying the screenshot tests, it feels weird to run a test that doesn't actually check anything. With that in `gradle.properties` we can also remove the explicit `verifyRoborazziDemoDebug` that we do above with the `TODO` and just use `testDemoDebugUnitTest`?

Reply:
I don't think we can, because steps `checkfork_screenshots` and `screenshotsrecord` depend on whether `screenshotsverify` passes. Also, that step has `continue-on-error: true` so a bad unit test would be ignored.